README.md

```diff
@@ -28,14 +28,22 @@ Our [`python-plugwise`](https://github.com/plugwise/python-plugwise) python modu
 
 # Changelog
 
-## NEW MAR 2022 [0.21.5]
+## MAR 2022 [0.22.0]
+- Smile:
+  - Link to plugwise v0.17.0 - https://github.com/plugwise/python-plugwise/releases/tag/v0.17.0
+  - Rework select.py, add selector for the heating/cooling-system regulation mode.
+  - Add number.py (and tests), provide an option for changing the maximum boiler (water) temperature setpoint.
+  - Remove climate schedule attributes, moved to the options-attribute of the select schedule entity.
+  - Implement fix for the sticking HA persistent notifications
+
+## MAR 2022 [0.21.5]
  - Smile:
   - **BREAKING** improved naming of the locally present outdoor temp sensor connected to the OpenTherm device: `outdoor_air_temperature`. The former `sensor.opentherm_outdoor_temperature` is now visible as `sensor.opentherm_outdoor_air_temperature`. Use of the zipcode based `outdoor_temperature` has not changed.
-  - **POTENTIALLY BREAKING** for consistency renamed schema to schedule, i.e. the attributes will now be `available_schedules` and `selected_schedule`
+  - **POTENTIALLY BREAKING** for consistency renamed schema to schedule, i.e. the attributes will now be `available_schedules` and `selected_schedule`.
```

Comment:
Do we really want to change them or obsolete them like suggested with Core? And/or we remove them from core here, right?